hub/core/query/query.py

```diff
@@ -243,7 +247,11 @@ def size(self):
         return self.arr.size  # type: ignore
 
     def contains(self, o: object) -> bool:
-        return any(self.arr == o)  # type: ignore
+        v = self.numpy() == o
+        if isinstance(v, np.ndarray):
+            return v.any()  # type: ignore
+        else:
+            return v
```

Comment:
in which case is this branch reached?

Reply:
In case, when comparison between `scallar == scalar`, result is just a boolean